static/css/main.css

```diff
@@ -1,31 +1,13 @@
 body {
 	margin: 0;
 	font-size: 140%;
-	background-color: #333;
```

Comment:
On `http://openfisca.org/en/tracking/`, the tracking status background/iframe is also white. Update it to a light grey or any color that looks good to you?